src/pkg/services/m365/api/events.go

```diff
@@ -289,63 +297,223 @@ func (c Events) GetItem(
 		}
 	)
 
-	event, err = c.Stable.
+	// Beta endpoint helps us fetch the event exceptions, but since we
+	// don't use the beta SDK, the exceptionOccurrences and
+	// cancelledOccurrences end up in AdditionalData
+	// https://learn.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-beta#properties
+	rawURL := fmt.Sprintf(eventExceptionsBetaURLTemplate, userID, itemID)
+	builder := users.NewItemEventsEventItemRequestBuilder(rawURL, c.Stable.Adapter())
```

Comment:
Wondering if I should switch from `Stable` to `LargeItem` as this now can contain a lot of data if a lot of items are modified.

Reply:
The primary difference is a 1 hour timeout vs an unlimited timeout.  Mail attachments have a pretty small size limit, iirc.  I imagine the stable client should be plenty here.